packages/ses/src/error/console.js

```diff
@@ -191,9 +196,9 @@ const makeCausalConsole = (baseConsole, loggedErrorHandler) => {
    * @returns {any}
    */
   const extractErrorArgs = (logArgs, subErrorsSink) => {
-    const argTags = logArgs.map(arg => {
-      if (arg instanceof Error) {
-        subErrorsSink.push(arg);
+    const argTags = arrayMap(logArgs, arg => {
+      if (arg instanceof FERAL_ERROR) {
```

Comment:
```suggestion
      if (isError(arg)) {
```
assuming the previous suggestion of adding an `isError` to commons.